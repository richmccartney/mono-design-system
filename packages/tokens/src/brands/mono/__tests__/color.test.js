const colorTokens = require('../color.json');

describe('color tokens', () => {
  test(': matches snapshot', () => {
    expect(colorTokens).toMatchSnapshot();
  });
});
