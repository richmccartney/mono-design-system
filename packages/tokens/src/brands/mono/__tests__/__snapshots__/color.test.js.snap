// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`color tokens : matches snapshot 1`] = `
Object {
  "color": Object {
    "brand": Object {
      "primary": Object {
        "value": "{color.bright-green.value}",
      },
      "secondary": Object {
        "value": "{color.forest-green.value}",
      },
    },
    "bright-green": Object {
      "value": "#9BE36E",
    },
    "forest-green": Object {
      "value": "#203207",
    },
    "forest-green-light": Object {
      "value": "#4D5B39",
    },
  },
}
`;
